src/oss/confluent.sh

```diff
@@ -1741,6 +1741,11 @@ invalid_requirement() {
 
 requirements
 
+echo "WARNING: This Command Line tool is not intended for production use.
```

Comment:
I suggest, either calling it `CLI` with caps or `command-line tool`

Reply:
Also, `WARNING` doesn't seem appropriate here. `Note:` maybe?

Reply:
@kkonstantine not sure what you mean by "passing a couple of examples... output"?

Reply:
What's printed when you run a few commands such as `confluent start` of `confluent stop` or `confluent load s3-sink`

Reply:
The raw output on the terminal